docs/developer/accessibility.md

```diff
@@ -0,0 +1,35 @@
+# Accessibility
+
+FAST-DNA has worked diligently from the ground up to build a component framework that meets all WCAG 2.0 AA standards. Every new component or website revision involves a rigorous approval process to validate and document compliance.
+
+Active collaboration occurs with both Microsoft’s legal, accessibility, and marketing teams to deliver inclusive and accessible experiences.
+
+To learn more about Microsoft's commitment visit their [accessibility](https://www.microsoft.com/en-us/accessibility) website.
```

Comment:
We should rework this detail regarding accessibility as I don't think that the goals outlined are an accurate representation; if they are, we don't meet them as components in nearly all cases are not "accessible" but support accessibility.

Reply:
reworded.